cloudinit/config/cc_ansible.py

```diff
@@ -6,61 +6,23 @@
 import sys
 import sysconfig
 from copy import deepcopy
-from textwrap import dedent
 from typing import Optional
 
 from cloudinit import subp
 from cloudinit.cloud import Cloud
 from cloudinit.config import Config
-from cloudinit.config.schema import MetaSchema, get_meta_doc
+from cloudinit.config.schema import MetaSchema
 from cloudinit.distros import ALL_DISTROS, Distro
 from cloudinit.settings import PER_INSTANCE
 from cloudinit.util import Version, get_cfg_by_path
 
 meta: MetaSchema = {
     "id": "cc_ansible",
-    "name": "Ansible",
-    "title": "Configure ansible for instance",
     "frequency": PER_INSTANCE,
     "distros": [ALL_DISTROS],
     "activate_by_schema_keys": ["ansible"],
-    "description": dedent(
-        """\
-        This module provides ``ansible`` integration for
-        augmenting cloud-init's configuration of the local
-        node.
+}  # type: ignore
```

Comment:
I'm assuming that at the end of all of the conversions you plan to update the MetaSchema and remove this comment?

Reply:
Correct, once the final branch drops all `examples`, `title`  and `name` keys we can update MetaSchema and ensure typing is upheld.